<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-F4K1FN9" failures="1" tests="2" name="Test" time="30.756" errors="0" timestamp="2023-05-05T21:39:39 IST">
  <testcase classname="TestCases.VerifyLogin" name="invalidLogin" time="1.93">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	error message is not valid expected [Please check your username. If you still can&amp;apos;t log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
error message is not valid expected [Please check your username. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at Resources.CommonMethods.handleAssertions(CommonMethods.java:18)
at TestCases.VerifyLogin.invalidLogin(VerifyLogin.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
  <testcase classname="TestCases.VerifySignUp" name="signup" time="9.518"/>
</testsuite> <!-- Test -->
